formal-resume/resume_english.tex

```diff
@@ -22,42 +22,42 @@
 
 \basicInfo{
   \email{Kenji.Mouri@outlook.com} \textperiodcentered\
-  \homepage{https://mouri.moe}
+  \homepage{https://mouri.moe} \textperiodcentered\
   \github[MouriNaruto]{https://github.com/MouriNaruto}
 }
 
 \section{Introduction}
 \begin{itemize}
-  \item Hi, I'm Kenji Mouri. But my legal name is Qi Lu which is used only when offline. MouriNaruto, Mouri\_Naruto and Mouri are my typical usernames.
-  \item I'm interested in writing the most compact implementations by using the least syntaxes and third-party libraries. Also, I am desired to make more friends which can talk something about technology.
-  \item I have created and maintained several open-source projects mostly written in C/C++ at night since 2014.
-  \item I'm also a proud Microsoft MVP with Developer Technologies and Windows Development award categories.
-  \item I hope my next job can do something about system or bare-metal related software development and maintenance.
+  \item Hi, I am Kenji Mouri. My legal name is Qi Lu which is used only when offline. MouriNaruto, Mouri\_Naruto and Mouri are my typical usernames.
+  \item I am passionate about developing highly efficient software implementations with minimal syntax and reliance on third-party libraries.
+  \item I have created and maintained several open-source projects written predominately in C and C++ since 2014.
+  \item I am also a proud Microsoft MVP in the Developer Technologies and Windows Development award categories.
+  \item My goal for my next role is system-level software development and maintenance.
 \end{itemize}
 
 \section{Professional Experience}
 
 \datedline{\href{https://live.qq.com}{\textbf{Qi'e TV, Tencent}} (C++ Software Development Engineer)}{Dec 2020 -- Present}
 \begin{itemize}
-  \item Lead the development of the QieLive 2.x Series from scratch to release, adapt for Windows 7 Service Pack 1 or later
-  \item Building customized FFmpeg, MSBuild and Qt6 toolchains for QieLive
+  \item Lead the development of the QieLive 2.x Series from conception to release, adapted for Windows 7 Service Pack 1 or later
+  \item Built customized FFmpeg, MSBuild, and Qt6 toolchains for QieLive
 \end{itemize}
 
 \section{Open Source Experience}
 
 \datedline{\href{https://github.com/M2Team}{\textbf{M2-Team}} (Owner)}{Jun 2015 -- Present}
```

Comment:
It's great to list these. You have to have some idea of the number of downloads and users. You should add that data, e.g. "With over X downloads." It shows not only development but adoption, that you deliver products that people enjoy and use.

Reply:
Actually, the downloads and users may not seem popular. But I know the winget collects that. https://github.com/microsoft/winget-pkgs/blob/master/manifests/t/Tencent/QieLive/2.0.11/Tencent.QieLive.locale.en-US.yaml

Kenji Mouri

Reply:
I will keep that to the original state which I merged because it doesn't have more highlights at the current stage.